backend/.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+profile = black
```

Comment:
Missing EOF at the end of a line.
It's quite important for type checkers (not added here in this project), but it's worth to always keep for two reasons:
1) type checkers
2) GitHub summary does not show additional line